src/test/java/com/wikia/webdriver/testcases/adstests/TestNoAdsLoggedInUsers.java

```diff
@@ -35,11 +35,9 @@ public TestNoAdsLoggedInUsers(String wikiName, String path) {
   }
 
   private void loginSteps() {
```

Comment:
It came from another person, but I will change it for you and for clearness. :dancers: